topics/proteomics/tutorials/metaquantome-taxonomy/tutorial.md

```diff
@@ -0,0 +1,404 @@
+---
+layout: tutorial_hands_on
+
+title: "metaQuantome 3: Taxonomy"
+zenodo_link: "https://doi.org/10.5281/zenodo.4110725"
+questions:
+ - "How do I look at differentially expressed taxa?"
+ - "Can we get top 5 or 10 taxa present in the sample?"
+ - "How can I visualize my results?"
+objectives:
+  - "A taxonomy and quantitational analysis of metaproteomic mass spectrometry data."
+time_estimation: "1h"
+key_points:
+  - "Statistical analysis of the differentially expressed taxa in the metaproteomics sample and their abundance."
```

Comment:
could you change the keypoints again to be take-home messages / conclusions from the tutorial rather than a description of what was done? And same for the function tutorial. (Just like you did with the first metaQuantome tutorial, those were good!)

I think that is the last change needed to merge this, awesome job! :+1: